src/components/ShelterListItem/ShelterListItem.tsx

```diff
@@ -57,21 +64,41 @@ const ShelterListItem = (props: IShelterListItemProps) => {
         {data.address}
       </h6>
       {data.shelterSupplies.length > 0 && (
-        <div className="flex flex-col gap-3">
-          <Separator className="mt-2" />
-          <p className="text-muted-foreground text-sm md:text-lg font-medium">
-            Precisa urgente de doações de:
-          </p>
-          <div className="flex gap-2 flex-wrap">
-            {tags.map((s, idx) => (
-              <Chip
-                className={getSupplyPriorityProps(s.priority).className}
-                key={idx}
-                label={s.supply.name}
-              />
-            ))}
+        <>
+          <div className="flex flex-col gap-3">
+            <Separator className="mt-2" />
+            <p className="text-muted-foreground text-sm md:text-lg font-medium">
+              Status voluntários:
+            </p>
+            <div className="flex gap-2 flex-wrap">
+              {volunteerTags.length == 0 ?
+                <p> Não informado. <i> (Pode ser adicionado ao clickar no abrigo) </i></p>
```

Comment:
Olá pessoal, vi [esse post no linkedin](https://www.linkedin.com/posts/rhuam_bom-dia-rede-precisamos-de-devs-na-segunda-activity-7194308980054118401-On4d?utm_source=share&utm_medium=member_desktop) e vim pra ajudar com a revisão do código.

```suggestion
                <p> Não informado. <i> (Pode ser adicionado ao clicar no abrigo) </i></p>
```